### PR TITLE
Add secure_verify test

### PR DESCRIPTION
**Pull request by asvrada**
Adding a unit test to check if `oe_verify_evidence` works inside enclave

Signed-off-by: Zijie Wu <zijiewu@microsoft.com>

### CODE REVIEW

**Comment by asvrada** [comment]
bors try

File: tests/secure_verify/secure_verify.edl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+    include "openenclave/bits/types.h"
+    include "openenclave/bits/sgx/sgxtypes.h"
+
+    trusted {
+
+        public oe_result_t verify_plugin_evidence(
+          [in] const oe_uuid_t* format_id,
+          [in, count=evidence_size,] uint8_t* evidence_in,
```
Comment (mingweishih) [inline tests/secure_verify/secure_verify.edl:22]:
```suggestion
          [in, count=evidence_size] uint8_t* evidence_in,
```

File: tests/secure_verify/host/host.cpp [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+
+done:
+    // deallcate resources
+    if (cert)
+        free(cert);
+
+    fflush(stdout);
+    return result;
+}
+#endif
```
Comment (mingweishih) [inline tests/secure_verify/host/host.cpp:422]: nit: new line

File: tests/secure_verify/host/host.cpp [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+            (tick2 - tick1));
+    else
+        printf(
+            "\nEvidence verification failed. Result = %s\n",
+            oe_result_str(result));
+
+    return ret;
+}
+
+#if 0 // TODO
```
Comment (mingweishih) [inline tests/secure_verify/host/host.cpp:290]: Maybe we can get rid of these disabled code for now.

**Review by mingweishih — APPROVED** [review]
LGTM. Can you squash the commits?

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try-

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try

**Comment by asvrada** [comment]
bors try

**Pull request merged**